docs/operator-guides/resumability.md

```diff
@@ -0,0 +1,85 @@
+---
+products: all
+---
+
+# Resumability & Resumable Full Refresh
+
+:::info
+
+This page describes a coming-soon feature. Resumable Full Refresh will be available as of Airbyte v1.0.0.
+
+:::
+
+## Resumability
+
+Airbyte strives to create resilient syncs that can handle many types of issues (for example, networking issues, flaky source APIs or under provisioned
+destination servers). Whenever possible, Airbyte syncs are resumable. “Resumable” in this context means that if something goes wrong in the first attempt
+of your sync, we will immediately try again with a subsequent attempt - in fact, Airbyte will keep retrying as long as the source is producing new records
+and the destination can commit them. In order to achieve this resumability, Airbyte may deliver a record more than once - **Airbyte is a deliver at-least-once system**.
+
+Consider a Postgres source. We could build a query that does `select * from users`, which would guarantee that each row is emitted only once, but that would lead to some problems:
+
+- First, it would keep a long-lasting transaction lock on the users table for the whole duration of the sync which would slow down the source
```

Comment:
```suggestion
- First, it would keep a long-lasting transaction lock on the `users` table for the duration of the sync.  This would slow down the source.
```